docs/Brew-Livecheck.md

```diff
@@ -75,63 +75,81 @@ We sometimes make this more explicit to exclude unwanted matches. URLs with a pr
 ### Version directories
 
 ```ruby
-  livecheck do
-    url "https://www.example.com/releases/example/"
-    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
-  end
+livecheck do
+  url "https://www.example.com/releases/example/"
+  regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
+end
 ```
 
 When checking a directory listing page, sometimes files are separated into version directories (e.g. `1.2.3/`). In this case, we must identify versions from the directory names.
 
 ### Git tags
 
 ```ruby
-  livecheck do
-    url :stable
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
+livecheck do
+  url :stable
+  regex(/^v?(\d+(?:\.\d+)+)$/i)
+end
 ```
 
 When the `stable` URL uses the `Git` strategy, the regex above will only match tags like `1.2`/`v1.2`, etc.
 
 If tags include the software name as a prefix (e.g. `example-1.2.3`), it's easy to modify the regex accordingly: `/^example[._-]v?(\d+(?:\.\d+)+)$/i`
 
-### `PageMatch` `strategy` block
+### `strategy` blocks
+
+If the upstream version format needs to be manipulated to match the formula/cask format, a `strategy` block can be used instead of a `regex`. 
+
+#### `PageMatch` `strategy` block
+
+In the example below, we're converting a date format like `2020-01-01` into `20200101`.
 
 ```ruby
-  livecheck do
-    url :homepage
-    regex(/href=.*?example[._-]v?(\d{4}-\d{2}-\d{2})\.t/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| match&.first&.gsub(/\D/, "") }
-    end
+livecheck do
+  url :homepage
+  strategy :page_match do |page|
+    page.scan(/href=.*?example[._-]v?(\d{4}-\d{2}-\d{2})\.t/i)
+        .map { |match| match&.first&.gsub(/\D/, "") }
   end
+end
 ```
 
-When necessary, a `strategy` block allows us to have greater flexibility in how upstream version information is matched and processed. Currently, they're only used when the upstream version format needs to be manipulated to match the formula/cask format. In the example above, we're converting a date format like `2020-01-01` into `20200101`.
-
 The `PageMatch` `strategy` block style seen here also applies to any strategy that uses `PageMatch` internally.
 
-### `Git` `strategy` block
+#### `Git` `strategy` block
+
+A `strategy` block for `Git` is a bit different, as the block receives an array of tag strings instead of a page content string. Similar to the `PageMatch` example, this is converting tags with a date format like `2020-01-01` into `20200101`.
```

Comment:
I have the commentary after the code block in the other examples, so we should standardize on having it either before or after for all the examples.